### DISCUSSION TITLE
youtube-bb-utility/download_detection.py //problem

### DISCUSSION THREAD

**Discussion by way790110** [STVIR/pysot#302] [General]
I executed yt_bb readme.md
line 20 : python download_detection.py ../ 12 , exist error.
Is it normal ?

![test](https://user-images.githubusercontent.com/33109449/75625646-0f4f5e00-5bfb-11ea-84b0-9246e2be7a65.jpg)

**Comment by ny222333** [comment]
Did you download successfully?I am downloading it while I find it is much more than 400G.

**Reply by HOLYlmx** [reply]
大佬 请问有未crop的Yt-bb数据资源吗 求分享~😊

**Comment by way790110** [comment]
@ny222333 I am trying.
I use 1T disk ,I think 1T is enough.
Is Error a normal situation?

**Comment by ny222333** [comment]
sorry,I did't have this error! Can we communicate using WeChat, I have a lot of questions to ask for help. hhhh~

**Comment by way790110** [comment]
@ny222333 ok
What is your WeChat ID?

**Comment by ny222333** [comment]
Ny08250919



At 2020-03-02 16:52:47, "way790110" <notifications@github.com> wrote:

@ny222333 ok
What is your WeChat ID?

**Comment by cappuccino19** [comment]
this is not available. try to download from Baidu disk. the link is in the other page of pysot.

**Comment by ny222333** [comment]
But   the baidu disk  link  is  invalid

在 2020-03-03 23:04:20，cappuccino19 <notifications@github.com> 写道：


this is not available. try to download from Baidu disk. the link is in the other page of pysot.

**Comment by cappuccino19** [comment]
Sorry, the database is huge, I can't transform to you. and  I still try to use this database too.

**Comment by way790110** [comment]
@cappuccino19 
Which operating system do you use ?

**Comment by cappuccino19** [comment]
linux ubuntu

On Wed, Mar 4, 2020 at 9:21 AM way790110 <notifications@github.com> wrote:

> @cappuccino19 <https://github.com/cappuccino19>
> Which operating system do you use ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/STVIR/pysot/issues/302?email_source=notifications&email_token=AEYWHWMCNARXN7LL4FDFBN3RFYFQLA5CNFSM4K7EM7WKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOENWZYIQ#issuecomment-594385954>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEYWHWJAAFBNC2G7SKQM53DRFYFQLANCNFSM4K7EM7WA>
> .
>


-- 
GAO Bin
Vision Lab, ISEN-Brest
Batiment KC, C104, 2 Rue des Archives, 29200 Brest    Post Code:   29219,
France

**Comment by way790110** [comment]
@cappuccino19 
Did you create new env(python=2.7) to download youtube-bb data?
Could you tell me what version you download about youtube-bb-utility/requirement.txt?
I used lastest python package then I get error.
I changed pandas, moviepy and imageio version then I still get Error like picture(Top).

**Comment by cappuccino19** [comment]
感觉就是没有下载授权了，跟系统和软件没关系。我微信cappu19

On Thu, Mar 5, 2020 at 5:10 AM way790110 <notifications@github.com> wrote:

> @cappuccino19 <https://github.com/cappuccino19>
> Did you create new env(python=2.7) to download youtube-bb data?
> Could you tell me what version you download about
> youtube-bb-utility/requirement.txt?
> I used lastest python package then I get error.
> I changed pandas, moviepy and imageio version then I still get Error like
> picture(Top).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/STVIR/pysot/issues/302?email_source=notifications&email_token=AEYWHWMQSEGFE5Y73ZA6GMDRF4Q4VA5CNFSM4K7EM7WKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEN3UIHA#issuecomment-595018780>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEYWHWPZSBNGQSN33P6LFXDRF4Q4VANCNFSM4K7EM7WA>
> .
>


-- 
GAO Bin
Vision Lab, ISEN-Brest
Batiment KC, C104, 2 Rue des Archives, 29200 Brest    Post Code:   29219,
France

**Comment by ZhiyuanChen** [comment]
I'll see if I can upload the YBB dataset to somewhere accessible in mainland China. 
But it has to wait until I get back to Australia. 
In the meantime, I couldn't provide more help, sorry.

**Comment by way790110** [comment]
@zyc-ai  When do you go back to Australia?

**Comment by ZhiyuanChen** [comment]
It mainly depends on when will the travel restrictions be cancelled and how the pandemic goes in Australia. 
It could take a few months or more.
